marker_dict: DICT_4X4_50
marker_length: 0.01
predefined_marker_path: 
camera_info_topic: /camera/aligned_depth_to_color/camera_info
color_topic: /camera/color/image_raw
camera_link: camera_link